server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -156,7 +154,14 @@ public HttpResponse grantToken(OAuthTokenExchangeRequest request) {
             .tokenType(AuthTypes.BEARER)
             .build();
 
-    return HttpResponse.ofJson(response);
+    Cookie cookie = Cookie.secureBuilder("UC_TOKEN", accessToken).path("/").build();
```

Comment:
@jamieknight-db might comment on any specific cookie settings she needs for the UI, looks like `secureBuilder()` sets `secure(true).httpOnly(true).sameSite("Strict")` which feels right.

Reply:
I might be misunderstanding, but I don't plan to utilize the cookie/token at all on the FE. I plan to call a `currentUser` endpoint and a `login` endpoint, both of which will hopefully send me back a `user` object upon success (if auth is enabled).

Reply:
Sorry yeah thanks for the reminder, yeah you'll let the browser take care of it. That does remind me, we should probably set a cookie timeout, say 5 days or something like that?

Reply:
@creechy @jamieknight-db  what is the recommended timeout value?

Reply:
There isn't really a recommended timeout value, it just depends on the use case. I think @creechy's suggestion of 5 days or 7 days is good 👍  thank you!

Reply:
@jamieknight-db  yeah we have kept it as default 5 days and this value is configurable(can be changed in server.properties)